scripts/check-events

```diff
@@ -308,7 +308,7 @@ $all_events = Hash::new { |hash, key| hash[key] = Event::new(key) }
 error_count = 0
 
 # Scan sources and build internal structures
-Find.find('.') do |path|
+Find.find('./src') do |path|
```

Comment:
This probably needs to include `./lib` as well.

Reply:
There are some internal events defined in `lib/vector-common/src/internal_event`.

Reply:
Buffers also have internal events defined - should definitely do `src` and `lib`